scala/ppml/src/main/scala/com/intel/analytics/bigdl/ppml/attestation/BigDLRemoteAttestationService.scala

```diff
@@ -33,80 +37,220 @@ import akka.util.Timeout
         opt[Boolean]('s', "httpsEnabled")
-          .text("httpsEnabled")
+          .text("Whether enable https, default is false")
           .action((x, c) => c.copy(httpsEnabled = x))
         opt[String]('t', "httpsKeyStoreToken")
-          .text("httpsKeyStoreToken")
+          .text("KeyS toreToken of https, default is token")
           .action((x, c) => c.copy(httpsKeyStoreToken = x))
-        opt[String]('h', "httpsKeyStorePath")
-          .text("httpsKeyStorePath")
+        opt[String]("httpsKeyStorePath")
+          .text("KeyStorePath of https, default is ./key")
           .action((x, c) => c.copy(httpsKeyStorePath = x))
+        opt[String]('k', "secretKey")
+          .text("Secret Key to encrypt and decrypt BigDLRemoteAttestation data file")
+          .action((x, c) => c.copy(secretKey = x))
+        opt[String]('b', "basePath")
+          .text("Diretory for data files of BigDL Remote Attestation Service")
+          .action((x, c) => c.copy(basePath = x))
+        opt[String]('e', "enrollFilePath")
+          .text("Path of base data file to save user information, "
+            + "default is ./BigDLRemoteAttestationService.dat")
+          .action((x, c) => c.copy(enrollFilePath = x))
+        opt[String]('o', "policyFilePath")
+          .text("Path of policy data file, default is ./BigDLRemoteAttestationServicePolicy.dat")
+          .action((x, c) => c.copy(policyFilePath = x))
     }
     val params = cmdParser.parse(args, CmdParams()).get
+    val secretKey = params.secretKey
+    val enrollFilePath = params.basePath + "/" + params.enrollFilePath
+    val policyFilePath = params.basePath + "/" + params.policyFilePath
+    val userContent = Await.result(FileEncryptUtil.loadFile(enrollFilePath, secretKey), 5.seconds)
+    userMap = AttestationUtil.stringToMap(userContent)
+    val policyContent = Await.result(FileEncryptUtil.loadFile(policyFilePath, secretKey), 5.seconds)
+    policyMap = AttestationUtil.stringToMap(policyContent)
+
+    val t = new Thread {
+      override def run(): Unit = {
+        while (true) {
+          Thread.sleep(30 * 1000)
+          FileEncryptUtil.saveFile(enrollFilePath,
+            AttestationUtil.mapToString(userMap), secretKey)
+          FileEncryptUtil.saveFile(policyFilePath,
+            AttestationUtil.mapToString(policyMap), secretKey)
+        }
+      }
+    }
+    t.start()
 
     val route: Route =
         get {
           path("") {
             val res = s"Welcome to BigDL Remote Attestation Service \n \n" +
+            "enroll an account like: " +
+            "GET <bigdl_remote_attestation_address>/enroll \n" +
+            "registe a policy like: " +
+            "POST <bigdl_remote_attestation_address>/registePolicy \n" +
             "verify your quote like: " +
             "POST <bigdl_remote_attestation_address>/verifyQuote \n"
             complete(res)
+          } ~
+          path("enroll") {
+            val appID = UUID.randomUUID.toString
```

Comment:
enroll and registePolicy should be methods of this class. That will simplify your compare code and make method implementation more clear.